docs/2_4_common_schema.adoc

```diff
@@ -1703,11 +1715,19 @@ _The ordering of the variables in this list is defined by the exporting tool.]_
 For Co-Simulation, elements <<EventIndicator>> are ignored.
 _[If an FMU supports both Model Exchange and Co-Simulation, then the <<EventIndicator>> elements might be present, since it is needed for Model Exchange.]_
 
+|`Clock`
+|
+[[ClockElement,`<Clock>`]]
+List of all clocks and their clocked variables.
+All variables referenced by these lists must have <<variability,`variability == discrete`>>.
+A clocked variable my depend on multiple clocks and may therefore be listed in multiple <<ClockElement>> elements.
+_[More rigorous importers requiring a variable to be dependent on a single clock can reject FMUs violating this restriction.]_
+
 |====
 
-Elements <<Output>>, <<StateDerivative>>, <<DiscreteTimeState>> and <<InitialUnknown>> have the following attributes:
+Elements <<Output>>, <<StateDerivative>>, <<ClockedState>>, <<InitialUnknown>>, <<EventIndicator>> and <<ClockElement>> have (partially) the following attributes:
 
-.<<Output>>, <<StateDerivative>>, <<DiscreteTimeState>> and <<InitialUnknown>> attribute details.
+.<<Output>>, <<StateDerivative>>, <<ClockedState>>, <<InitialUnknown>>, <<EventIndicator>> and <<ClockElement>> attribute details.
```

Comment:
I cant really see the tables, and dot fully understand the changes so maybe I am misudnerstanding

But i don't think we should mix up the clocks with the rest and overload the dependecies attribute, whileremove dependecyKind.

Better to be clear. make a separate small table
Value reference ....
Clocked variables, list of value references to all variables that depends on this clock. Only variables with variability discrete can be listed. 
An empty list means no variables belongs to a partition of this clock.
If attribute is missing then all variables with variability discrete is assumed to depend on this clock.

Reply:
Skip the last comment
@chrbertsch helped me get the build of this comit.

So this part is fine.
I would still prefer that we have a separate table for clocks instead of merging it with the other knowns, as this is different things and aren't really related. I think it would be much cleaner.

However this is a preference from me not a requirement.

Thanks for doing the work =)

Reply:
If we still want to change this, we can do so later, for now, let`s close this part of the conversation.